README.md

```diff
@@ -1 +1,136 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello-microverse] <a name="Hello-microverse"></a>
+
+**[Hello-microverse]** this was mini-projects that upskill to create a feature branch and test linters in order to have professional-looking code.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Creating your first "Hello-microverse" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-A Git hub account
+<br>
+-Node JS
+<br>
+-Visual Studio Code or any other code editor of your preference
+<br>
+-web browser
+
+### Setup
+
+Clone this repository to your desired folder:<br>
+git@github.com:brighton0102/Hello.git
+
+## Install Linters
+you can use 
+
+Github
+VS code
+so that you can access the code
+
+To run the project, execute the following commands
+
+Linters
+git commands
+
+### Run test
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: (https://github.com/brighton0102)
+- Twitter: (@brighton_ui)
+- LinkedIn: (in/brighton-mtandi-976615267)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions and feature request are welcome.
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+      This project will not have any additional features.
+
+
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, kindly leave a comment below and share it with other web developers and lets enjoy this journey together.
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="Microverse Inc."></a>
+
+I am thankful to be part of the less than one percent of the student's who get enrolled at microverse.
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+  Do I install linters on each and every project at microverse?
+
+  - [Answer_1]
+    Yes you have to use linters for each and every project at microverse.
+
+- **[Question_2]**
+    Should i always create a feature branch when working on projects?
+
+  - [Answer_2]
+    Yes you have to create a feature branch for your projects
```

Comment:
- [ ] Kindly remove the `FAQ` section if you don't have the faqs because it makes your `README.md` file look unprofessional. 

**FACT**: A *FAQ* is a document, in question and answer format, that introduces newcomers to a topic or answers common questions.